$MirOS: ports/mailnews/exim/patches/patch-OS_Makefile-OpenBSD,v 1.1.7.1 2005/03/18 15:49:34 tg Exp $
--- OS/Makefile-OpenBSD.orig	Tue Apr  4 13:59:41 2006
+++ OS/Makefile-OpenBSD	Wed Apr 26 14:45:42 2006
@@ -2,9 +2,8 @@
 
 # Exim: OS-specific make file for OpenBSD
 
-CHOWN_COMMAND=/usr/sbin/chown
-CHGRP_COMMAND=/usr/sbin/chgrp
-CFLAGS=-O2 -Wall
+CHOWN_COMMAND=/sbin/chown
+CHGRP_COMMAND=/bin/chgrp
 
 LIBS=-lm
 
